packages/cli/__tests__/__fixtures__/kuai-config-case/kuai.config2.ts

```diff
@@ -0,0 +1,13 @@
+import { task, KuaiConfig } from '@kuai/core';
+
+task('demo-task2')
+  .addParam('paramA', 'a custom param', 'default value')
+  .setAction(() => {
+    console.log('demo-task2');
+  });
+
+const config: KuaiConfig = {
+  kuai: '0.0.1',
+};
+
+export default config;
```

Comment:
`demo-task1` is similar to `demo-task2`, I think for the test case it only needs to leave one.

Reply:
These two files are designed to test that "different commands are loaded when different configuration files are specified"

Maybe it would be better to make the two files look a more different?

Reply:
Add some commands in a configuration, and test if it will load. I feel it's enough.